app/grandchallenge/reader_studies/forms.py

```diff
@@ -688,6 +689,54 @@ class Meta(PermissionRequestUpdateForm.Meta):
         model = ReaderStudyPermissionRequest
 
 
+class GroundTruthCopyAnswersForm(SaveFormInitMixin, Form):
+    user = ModelChoiceField(
+        queryset=get_user_model().objects.none(),
+        required=True,
+        help_text="Select a user who's answers will be copied.",
```

Comment:
who's is a contraction of who is

```suggestion
        help_text="Select a user whose answers will be copied.",
```